distribution/scripts/start-client.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Wrapper to force the client to wait for the backend to become available
+# Should only be useful during development
+set -euo pipefail
+
+sleepTime=8
```

Comment:
Not sure why this variable is necessary :man_shrugging:

Reply:
True, it could defined in the `if` scope below, if that is what you mean.

But it's needed since I use it in both the message and the `sleep $sleepTime` below on line 10.